packages/execution-environments/src/common/endowments/index.test.ts

```diff
@@ -75,18 +76,27 @@ describe('createEndowments', () => {
       Buffer,
       console,
       Math,
+      BigInt: expect.any(Function),
       setTimeout: expect.any(Function),
       clearTimeout: expect.any(Function),
       WebAssembly: { ...WebAssembly },
     });
 
     expect(endowments.wallet).toBe(mockWallet);
     expect(endowments.Buffer).toBe(Buffer);
-    expect(endowments.Math).toBe(Math);
     expect(endowments.console).toBe(console);
+    expect(endowments.Math).toBe(Math);
 
+    expect(endowments.BigInt).not.toBe(BigInt);
```

Comment:
How come `BigInt` is not the global `BigInt`? 🤔

Reply:
Because we `.bind` functions to the root realm global object: https://github.com/MetaMask/snaps-skunkworks/blob/6d1c289753d0075743e441f0996eeb32259365fc/packages/execution-environments/src/common/endowments/index.ts#L65-L69

Reply:
Can we add a comment to clarify this? It might be confusing as the test-case is grouped with other test-cases that are customized for platform reasons